apps/console/src/public/deployment.config.json

```diff
@@ -372,9 +372,7 @@
                 }
             },
             "applications": {
-                "disabledFeatures": [
-                    "applications.loginFlow.legacyEditor"
-                ],
```

Comment:
Hope reactivating the legacy editor is intentional :)

Reply:
Yeah this is already removed in default.json. This was missed. Verfied with brion.